packages/adapter/package.json

```diff
@@ -9,8 +9,9 @@
   ],
   "scripts": {
     "build": "yarn run clean && yarn run compile",
-    "clean": "rimraf -rf ./dist",
+    "clean": "rimraf -rf ./dist *.tgz",
```

Comment:
why not clean the whole dist?

Reply:
I am cleaning the whole `dist`. The `*.tgz` part is for cleaning the packages made with `yarn pack`.

Reply:
I would argue that yestedray it looked like `dist/*.tgz` but I probably should just buy glasses instead...